visma/matrix/checks.py

```diff
@@ -50,3 +50,17 @@ def multiplyCheck(matA, matB):
     if matA.dim[1] == matB.dim[0]:
         return True
     return False
+
+
+def isSquareMatrix(mat):
```

Comment:
This function is already present in the _Matrix_ class. You can update the `test_isSquareMatrix()` using the already defined class method.